doc/rados/operations/pools.rst

```diff
@@ -604,47 +604,38 @@ You may set values for the following keys:
                  from these replies. This approach sacrifices resources in 
                  exchange for better performance. This flag is supported only 
                  for erasure-coded pools.  :Type: Boolean :Defaults: ``0``
-
 .. _scrub_min_interval:
 
 .. describe:: scrub_min_interval
-
-   The minimum interval in seconds for pool scrubbing when
-   load is low. If it is 0, the value osd_scrub_min_interval
-   from config is used.
+   
+   :Description: Sets the minimum interval (in seconds) for pool scrubbing when load is low. If the default value of ``0`` is in effect, then the value of ``osd_scrub_min_interval`` from central config is used.
 
    :Type: Double
    :Default: ``0``
 
 .. _scrub_max_interval:
 
 .. describe:: scrub_max_interval
-
-   The maximum interval in seconds for pool scrubbing
-   irrespective of cluster load. If it is 0, the value
-   osd_scrub_max_interval from config is used.
+   
+   :Description: Sets the maximum interval (in seconds) for pool scrubbing regardless of cluster load. If the value of ``scrub_max_interval`` is ``0``, then the value ``osd_scrub_max_interval`` from central config is used.
 
    :Type: Double
    :Default: ``0``
 
 .. _deep_scrub_interval:
 
 .. describe:: deep_scrub_interval
-
-   The interval in seconds for pool “deep” scrubbing. If it
-   is 0, the value osd_deep_scrub_interval from config is used.
+   
+   :Description: Sets the interval (in seconds) for pool “deep” scrubbing. If the value of ``deep_scrub_interval`` is ``0``, the value ``osd_deep_scrub_interval`` from central config is used.
 
    :Type: Double
    :Default: ``0``
 
 .. _recovery_priority:
 
 .. describe:: recovery_priority
-
-   When a value is set it will increase or decrease the computed
-   reservation priority. This value must be in the range -10 to
-   10.  Use a negative priority for less important pools so they
-   have lower priority than any new pools.
+   
+   :Description: Setting this value adjusts a pool's computed reservation priority.  reservation priority. This value must be in the range ``-10`` to ``10``. Any pool assigned a negative value will be given a lower priority than any new pools, so users are directed to assign negative values to low-priority pools.
```

Comment:
@zdover23 - Delete the extra _ reservation priority._

Reply:
Good catch. Corrected.